README.md

```diff
@@ -14,7 +14,7 @@ git clone https://github.com/kubernetes-client/c
 CLIENT_REPO_ROOT=${PWD}/c
 
 # Install pre-requisites
-sudo apt-get install libssl-dev libcurl4-openssl-dev libwebsockets-dev uncrustify
+sudo apt-get install libssl-dev libcurl4-openssl-dev libwebsockets-dev uncrustify cmake g++
```

Comment:
A small suggestion: Since the client library is written for C programming, not just C++, it would be best to remove the dependency on `g++`.
Since this PR has already been merged, we may need another PR to remove it.
Sorry for the delay in my review.